fe/fe-core/src/main/java/com/starrocks/catalog/IcebergResource.java

```diff
@@ -64,6 +68,17 @@ protected void setProperties(Map<String, String> properties) throws DdlException
                     throw new DdlException(ICEBERG_METASTORE_URIS + " must be set in properties");
                 }
                 break;
+            case CUSTOM_CATALOG:
+                catalogImpl = properties.get(ICEBERG_IMPL);
+                if (StringUtils.isBlank(catalogImpl)) {
+                    throw new DdlException(ICEBERG_IMPL + " must be set in properties");
+                }
+                try {
+                    Thread.currentThread().getContextClassLoader().loadClass(catalogImpl);
```

Comment:
This is just to verify whether the jvm has loaded the custom class？ If fe restarts and the loadClass() method will not be called, Will it affect the loadCatalog later?

Reply:
Are there other considerations here? In fact, there is no need to restart fe by dynamically loading the custom jar.